### PR TITLE
Add product to favourites

### PR DESCRIPTION
**Pull request by ihorhladkov**

### CODE REVIEW

**Review by perfectAcedia — APPROVED** [review]
Take a look at my last PR and implement a local storage feature as well.

File: src/pages/FavouritesPage.tsx [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+  //       params: { page: 1, perPage: 8 },
+  //     })
+  //     .then((res) => {
+  //       setProducts(res.data.data);
+  //     })
+  //     .catch((e) => {
+  //       throw new Error(e);
+  //     });
+  // }, []);
   // #endregion
```
Comment (MaxSchmide) [inline src/pages/FavouritesPage.tsx:29]: please delete comments

File: src/hooks/useLocalStorage.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react';
+import { getInitialFavourites, useAppDispatch } from '../redux/';
+
+export const useInitFavourites = () => {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(getInitialFavourites());
+  }, []);
+};
```
Comment (MaxSchmide) [inline src/hooks/useLocalStorage.ts:10]: GJ, but, please, rename file to your hook name

**Review by MaxSchmide — CHANGES_REQUESTED** [review]

**Review by MaxSchmide — APPROVED** [review]
Looks good

**Pull request merged**